src/MSBuild/Resources/xlf/Strings.pl.xlf

```diff
@@ -178,6 +178,21 @@ Copyright (C) Microsoft Corporation. Wszelkie prawa zastrzeżone.
       LOCALIZATION: "MSBuild" should not be localized.
       LOCALIZATION: "-graphBuild" and "-graph" should not be localized.
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </note>
+      </trans-unit>
+      <trans-unit id="HelpMessage_37_LowPrioritySwitch">
+        <source>  -lowPriority[:True|False]
+                     Causes MSBuild to run at low priority.
+                     (Short form: -low)
```

Comment:
"Specifying /lowPriority is the same as specifying /lowPriority:true"

Reply:
Sure, thanks